emails/components/ProductDetails.tsx

```diff
@@ -3,42 +3,44 @@ import Logo from '../assets/logo.png';
 import { Button, Heading, Img, Link, Text } from '@react-email/components';
 import { PrivateCloudCreateRequestBody } from '@/schema';
 
-interface EmailProp {
-  data: PrivateCloudCreateRequestBody;
-}
-
-export default function Header({ data }: EmailProp) {
+export default function ProductDetails({
+  name,
+  description,
+  ministry,
+  po,
+  tl1,
+  tl2,
+}: {
+  name: String;
```

Comment:
I'm not certain if `String` with a capital 'S' is a valid TypeScript type. Additionally, let's try to minimize the usage of the `any` type wherever possible in the code below.